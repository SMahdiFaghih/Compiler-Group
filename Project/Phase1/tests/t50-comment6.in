/* this is a legal comment
/* this is too */
but no longer a comment
no longer a comment and a bunch of tokens follow */

not comment either
