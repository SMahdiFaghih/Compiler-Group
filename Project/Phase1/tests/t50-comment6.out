T_ID but
T_ID no
T_ID longer
T_ID a
T_ID comment
T_ID no
T_ID longer
T_ID a
T_ID comment
T_ID and
T_ID a
T_ID bunch
T_ID of
T_ID tokens
T_ID follow
*
/
T_ID not
T_ID comment
T_ID either
